.travis.yml

```diff
@@ -17,21 +17,27 @@ os:
 
 node_js:
   - "10"
-  - "10.15"
-  - "10.19"
   - "12"
   - "14"
-
 env:
-  # - TEST_DIR=client
+#   - TEST_DIR=client
   - TEST_DIR=server
 
-install:
-  - npm version
-  - npm install
-
-script:
-  - cd $TEST_DIR && npm install && npm test
+matrix:
+  include:
+    - os: windows
+      env: TEST_DIR=server YARN_GPG=no
+    - os: windows
+      node_js: "12"
+      env: TEST_DIR=server YARN_GPG=no
+    - os: windows
+      node_js: "14"
+      env: TEST_DIR=server YARN_GPG=no
+before_install:
+  - cd $TEST_DIR
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.22.4
+  - export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
Why is the font so strange?
Make sure to convert this to "simple" textfile before copy/pasting paths. Some tools on some operating systems would have issues to deal with this.

Reply:
The font looks normal on my machine:
<img width="699" alt="Screen Shot 2020-06-18 at 4 05 01 PM" src="https://user-images.githubusercontent.com/20048485/85080218-7890b800-b17d-11ea-99be-1fa498a6df5f.png">

not sure I totally understand what / how I should change this

Reply:
Funny. I see this on Github:
![image](https://user-images.githubusercontent.com/60901087/85080835-0e791280-b17f-11ea-8c41-3b9357eaa0c2.png)

I hope it's just a rendering issue for their new design!